voltixApp/VoltixApp/View Models/KeysignDiscoveryViewModel.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  KeysignDiscoveryViewModel.swift
+//  VoltixApp
+//
+
+import Foundation
+import Mediator
+import OSLog
+
+enum KeysignDiscoveryStatus {
+    case WaitingForDevices
+    case FailToStart
+    case Keysign
+}
+
+class KeysignDiscoveryViewModel: ObservableObject {
+    private let logger = Logger(subsystem: "keysign-discovery", category: "viewmodel")
+    var vault: Vault
```

Comment:
Can use let instead of var here for the declaration.

Reply:
need to change it at `setData` , so have to use var